src/main/resources/services/nameGraph/nameGraph.ts

```diff
@@ -48,6 +48,11 @@ function prepareResult(name: string): string {
 }
 
 function prepareGraph(name: string): Array<NameGraph> {
+  return nameGraphRepoExists() ? prepareGraphRepo(name) : prepareGraphDataset(name)
+}
+
+//TODO: Remove this when new repo nameGraph is created in production.
```

Comment:
Maybe we should keep this - it's good to have a fallback in case repo is missing.